tests/dualtor/test_standby_tor_upstream_mux_toggle.py

```diff
@@ -20,7 +20,7 @@
                             'run_garp_service', 'run_icmp_responder')
 ]
 
-PAUSE_TIME = 10
+PAUSE_TIME = 90
```

Comment:
the pause needs to take into effect only for cisco devices and time needs to be checked in a loop vs. always waiting for 90